feature-libs/organization/administration/core/store/reducers/index.ts

```diff
@@ -121,67 +122,69 @@ export function getReducers(): ActionReducerMap<OrganizationState> {
       approvalProcesses: StateUtils.entityLoaderReducer<B2BApprovalProcess[]>(
         ORG_UNIT_APPROVAL_PROCESSES_ENTITIES
       ),
-      users: StateUtils.entityLoaderReducer<ListModel>(
+      users: StateUtils.entityLoaderReducer<ListModel, any>(
         ORG_UNIT_ASSIGNED_USERS,
         orgUnitUserListReducer
       ),
-      addressList: StateUtils.entityLoaderReducer<ListModel>(
+      addressList: StateUtils.entityLoaderReducer<ListModel, any>(
         ADDRESS_LIST,
         orgUnitAddressListReducer
       ),
-      addressEntities:
-        StateUtils.entityLoaderReducer<Address>(ADDRESS_ENTITIES),
+      addressEntities: StateUtils.entityLoaderReducer<
+        Address,
+        AnyForUntypedForms
```

Comment:
Inline doc says to use `any` instead `AnyForUntypedForms`.  Could that be the case?

Reply:
I even don't know what is `AnyForUntypedForms`. I guess vs code suggests it, and I mistakenly selected it.